alacritty/src/display/mod.rs

```diff
@@ -412,6 +412,9 @@ impl Display {
         let (cell_width, cell_height) = compute_cell_size(config, &metrics);
 
         // Resize the window to account for the user configured size.
+        // In macOS, for keeping the window position after resizing
+        // at the top left window corner, it must be visible.
+        window.set_visible(true);
```

Comment:
Did you test this on other platforms? The current location of this call is where it is for a reason iirc.

Reply:
Unfortunately, I wasn't able to test the changes on other platforms. However, I've made adjustments to ensure platform-specific behavior. The call to set the window visible is now conditional based on the target platform, so it should maintain the expected behavior on macOS while not affecting other platforms. Is this approach acceptable to you?